contracts/TokenListManager.sol

```diff
@@ -2,96 +2,217 @@
 pragma solidity ^0.8.20;
 
 import {ITokenListManager} from "./ITokenListManager.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
 contract TokenListManager is ITokenListManager {
 
     // Storage the token address to whitelist or blacklist
     address[] private whitelistedTokensArray;
```

Comment:
Is this array required if different types tokens stored in separated array?

Reply:
Right, no need this array anymore, ill remove this array.

Reply:
Solved